lib/compiler/src/beam_ssa_alias.erl

```diff
@@ -740,6 +740,11 @@ aa_bif(Dst, tl, Args, SS, AAS) ->
 %% TODO: Ignored for now, as we don't track what's inside maps.
 %% aa_bif(_Dst, map_get, _Args, SS, _AAS) ->
 %%     SS;
+aa_bif(Dst, binary_part, Args, SS, _AAS) ->
+    %% bif:binary_part/3 is the only guard bif which could lead to
```

Comment:
Nitpick: The comment does not mention bif:binary_part/2 (which the code handles).

```suggestion
    %% bif:binary_part/2,3 is the only guard bif which could lead to
```

Reply:
True, I checked the implementation and saw that both variants of `binary_part` boiled down to the same function, never fed that back to the comment. Fixed (and updated the commit message accordingly).